examples/catalog/lib/main.dart

```diff
@@ -4,4 +4,11 @@
 
 import 'package:flutter/widgets.dart';
 
-void main() => runApp(const Center(child: const Text('flutter run -t lib/xxx.dart')));
+void main() => runApp(
```

Comment:
Use braces here instead of the function shorthand?

Reply:
Done.